galette/templates/default/elements/navigate.html.twig

```diff
@@ -1,25 +1,21 @@
 {% if navigate is defined and navigate|length != 0 %}
-    <nav class="ui mini inverted pagination menu right floated">
-        {% if navigate.prev is defined %}
-                <a
-                        href="{{ path_for(mpath, {"id": navigate.prev}) }}"
-                        class="item"
-                        title="{{ _T("Previous")|escape }}"
-                >
-                    <i class="step backward icon"></i>
-                    <span class="sr-only">{{ _T("Previous") }}</span>
-                </a>
-        {% endif %}
+    <nav class="ui mini inverted pagination menu">
+        <a
+            {% if navigate.prev is defined %}href="{{ path_for(mpath, {"id": navigate.prev}) }}"{% endif %}
```

Comment:
I removed the attribute by convenience.

Omiting the `href` only seems to be a problem on Firefox >= 41 if the link is supposed to be interactive : https://caniuse.com/mdn-html_elements_a

Otherwise, it should only be a problem when the href is involved in a js script.

Reply:
Ok :)